examples/system_metrics/collect_system_metrics.py

```diff
@@ -0,0 +1,12 @@
+import time
+
+import mlflow
+
+if __name__ == "__main__":
+    mlflow.enable_system_metrics_logging()
+    run = mlflow.start_run()
+    time.sleep(11)
+    mlflow.end_run()
```

Comment:
can we use `with mlflow.start_run()` here?

Reply:
mostly equivalent, but I am hoping to show that `end_run` needs to be called.

Reply:
> I am hoping to show that end_run needs to be called.

can you elaborate?

Reply:
ohh it's just in this example `mlflow.end_run()` is called, so people will follow the code.

Not a big deal, happy to change to `with mlflow.start_run()` if you prefer that one

Reply:
sg, done!